docs/getting_started/help.md

```diff
@@ -0,0 +1,9 @@
+# Getting help
+
+For assistance beyond this documentation, you can reach the Key4hep community via:
+
+- email: key4hep dash sw @ cern dot ch
+
+For issues or requests related to the documentation, please use the [bugtracker](https://github.com/key4hep/key4hep-doc/issues).
+
+For discussions or help with a specific Key4hep package, you can open a request on its repository's bug tracker. A [list of repositories](https://github.com/orgs/key4hep/repositories) is available on GitHub.
```

Comment:
I think we should also add a sentence here about opening an issue on key4hep-spack as a catch-all for issues where people either don't know the package (or don't want to find out). Effectively similar to what we are already doing in the setup script.